src/interfaces/source.ts

```diff
@@ -569,3 +569,11 @@ export class PrivateMessageReport {
   @Expose()
   updated: Option<string>;
 }
+
+export class Tagline {
+  id: number;
+  local_site_id:  number;
+  content: string;
+  published: string;
+  updated: string;
```

Comment:
updated is not a string, check your back-end struct again.

Reply:
Fixed, thanks!